src/console/Startup.cs

```diff
@@ -83,13 +83,15 @@ public virtual List<IEventConsumer> ResolveEventConsumers(IServiceProvider servi
                 var collectionContentFactory = serviceProvider.GetRequiredService<CollectionTemplateFactory<IContentTemplate, IContentTemplate>>();
                 var deviceDataNormalization = new Normalize.Processor(template, templateManager, collector, logger, collectionContentFactory);
                 eventConsumers.Add(deviceDataNormalization);
+                EventMetrics.ConnectorOperation = ConnectorOperation.Normalization;
```

Comment:
By setting `ConnectorOperation` here, is it set before any event metrics are logged?

Reply:
Yes, this basically the app startup code that is meant to set things up prior to the app actually running, so I don't believe any events wrt EventMetrics, would be logged prior to this.